src/histryExtension.ts

```diff
@@ -0,0 +1,114 @@
+import * as vscode from "vscode"
```

Comment:
This is mostly duplicate of `startExtension`, we should refactor this into one class.